FIrstModule/FIrstModule/Program.cs

```diff
@@ -1,63 +1,212 @@
-﻿void AddItem(string[,] items)
-{
-    Console.WriteLine("Input your item");
-    string item = Console.ReadLine().ToLower();
-    if (item != null)
-    {
-
-    }
-    else Console.WriteLine("String is null.");
-}
-void RemoveItem(string[,] items)
-{
-
-}
-void MarkAs(string[,] items)
-{
-
-}
-void Show(string[,] items)
-{
-
-}
-
-
+﻿
 Console.WriteLine("Hello, input the command for your tasks.");
-
+List<string> items = new List<string>() { "str", "add-item", "make a homework" };
 while (true)
 {
+    Console.WriteLine(Environment.NewLine);
     Console.WriteLine(">add-item");
     Console.WriteLine(">remove-item");
     Console.WriteLine(">mark-as");
     Console.WriteLine(">show");
     Console.WriteLine(">exit");
-
     Console.WriteLine("Input a command:");
     var command = Console.ReadLine();
-
+    Console.WriteLine(Environment.NewLine);
     switch (command)
     {
         case "add-item":
             {
-                AddItem(items);
-                break;
+                Console.WriteLine("Input a task:");
+                var task = Console.ReadLine();
+
+                if (!string.IsNullOrEmpty(task))
+                {
+                    if (!items.Any())
+                    {
+                        items.Add(task); break;
+                    }
+                    bool IsTask = true;
```

Comment:
Name your variables in a more clear way. Here you check whether the task exists. So name it taksExist.